### ISSUE TITLE
PyPi version differs from latest Github version

### ISSUE THREAD

**Issue by ariel-anieli** [ncclient/ncclient#226]
@leopoul,
The PyPi version differs from the latest Github version:

```
# pip show ncclient; echo "==";sed -ne '/new_/,$p' /usr/local/lib/python2.7/dist-packages/ncclient/xml_.py
Name: ncclient
Version: 0.5.3
Summary: Python library for NETCONF clients
Home-page: http://ncclient.org
Author: Shikhar Bhushan, Leonidas Poulopoulos, Ebben Aries
Author-email: shikhar@schmizz.net, lpoulopoulos@verisign.com, earies@juniper.net
License: Apache License 2.0
Location: /usr/local/lib/python2.7/dist-packages
Requires: setuptools, lxml, six, paramiko
You are using pip version 9.0.1, however version 9.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
==
new_ele = lambda tag, attrs={}, **extra: etree.Element(qualify(tag), attrs, **extra)

new_ele_ns = lambda tag, ns, attrs={}, **extra: etree.Element(qualify(tag,ns), attrs, **extra)

sub_ele = lambda parent, tag, attrs={}, **extra: etree.SubElement(parent, qualify(tag), attrs, **extra)
```
It needs an update:
```
# git clone --recursive https://github.com/ncclient/ncclient.git; echo "=="; sed -ne '/new_/,$p' ncclient/ncclient/xml_.py
Cloning into 'ncclient'...
remote: Counting objects: 4064, done.
remote: Compressing objects: 100% (29/29), done.
remote: Total 4064 (delta 18), reused 35 (delta 17), pack-reused 4015
Receiving objects: 100% (4064/4064), 2.77 MiB | 1.10 MiB/s, done.
Resolving deltas: 100% (2298/2298), done.
==
new_ele = lambda tag, attrs={}, **extra: etree.Element(qualify(tag), attrs, **extra)

new_ele_ns = lambda tag, ns, attrs={}, **extra: etree.Element(qualify(tag,ns), attrs, **extra)

sub_ele = lambda parent, tag, attrs={}, **extra: etree.SubElement(parent, qualify(tag), attrs, **extra)

sub_ele_ns = lambda parent, tag, ns, attrs={}, **extra: etree.SubElement(parent, qualify(tag, ns), attrs, **extra)
```

Could we do it?

Thanks,
Ariel

**Comment by einarnn** [comment]
Duplicate of #227. Looking into new release.
